3rd-party/cove/app/mainform.cpp

```diff
@@ -516,19 +521,25 @@ bool mainForm::performMorphologicalOperation(
 	Vectorizer::MorphologicalOperation mo)
 {
 	QString text;
+	bool transVectorizable {};
```

Comment:
This another example of the recent initialization style question. Maybe my concerns are more obvious here:
- In initialization, this style uses `{}` to express `false`.
- In assignement, a few lines later, `false` is used to express `false`.

Reply:
I see your point. I'll reconsider the use of the brace initialization, including zero init.